app/state/shieldsPanelState.ts

```diff
@@ -74,22 +75,22 @@ export const updateResourceBlocked: shieldState.UpdateResourceBlocked = (state,
 
   if (blockType === 'ads') {
     tabs[tabId].adsBlocked++
-    tabs[tabId].adsBlockedResources = [ ...tabs[tabId].adsBlockedResources, subresource ]
+    tabs[tabId].adsBlockedResources = unique([ ...tabs[tabId].adsBlockedResources, subresource ])
```

Comment:
Our block counts will not match the counts of resources we are showing, I think it might be better to match both.

Reply:
`Maybe set adsBlocked` to the length of adsBlockedResources after this line?